marketplaceItems/questionApp/appResources/appData/question_app.js

```diff
@@ -96,22 +108,67 @@ Tablet, Users, Vec3, Window */
             parentID: questionMark,
             materialData: JSON.stringify({
                 materials: {
-                    albedo: { red: 0, green: 255, blue: 0 },
-                    emissive: { red: 0, green: 1, blue: 0 }
+                    albedo: HIFI_CYAN,
+                    emissive: { red: 0, green: rgbValueToFloat(HIFI_CYAN.green), blue: rgbValueToFloat(HIFI_CYAN.blue) }
                 }
             })
         };
         questionMarkMaterial = Entities.addEntity(questionMarkMaterialProperties, 'avatar');
-        print("start growing");
-        growingInterval = Script.setInterval(function() {
-            var questionMarkDimensions = Entities.getEntityProperties(questionMark, 'dimensions').dimensions;
-            questionMarkDimensions = Vec3.multiply(questionMarkDimensions, GROWTH_RATIO);
-            Entities.editEntity(questionMark, { dimensions: questionMarkDimensions });
-            lastDimensions = questionMarkDimensions;
-            if (growingInterval && questionMarkDimensions.y > MAX_HEIGHT_M) {
-                print("stop growing");
-                Script.clearInterval(growingInterval);
-                growingInterval = null;
+        var toYellowPhase = true;
+        var colorChanging = true;
+        var growing = true;
+        changingInterval = Script.setInterval(function() {
+            if (!growing && !colorChanging) {
+                Script.clearInterval(changingInterval);
+                changingInterval = null;
+            }
+            try {
+                var materialData = JSON.parse(Entities.getEntityProperties(questionMarkMaterial, 'materialData').materialData);
+            } catch (err) {
+                print("ERROR:could not get material data");
+                return;
+            }
+            if (colorChanging && toYellowPhase) {
+                if (materialData.materials.albedo.green < HIFI_YELLOW.green) {
```

Comment:
won't this always be true? `materialData.materials.albedo.green` is going to be between 0 and 1, and then you're incrementing `materialData.materials.albedo.green` by `1`, and then you're dividing that result by 255...right? i don't think this is doing what you want it to be doing (i may be mistaken)

Reply:
albedo is a scale of 0-255, emissive is 0-1. I get all the color values in the 0-2555 scale, then convert to 0-1 scale for emissive.

Reply:
oh wow, it's so weird that albedo and emissive are on different scales like that. okay!

Reply:
yeah it is!